_posts/2017-05-15-rust-at-two-years.md

```diff
@@ -0,0 +1,356 @@
+[Oxidation]: https://wiki.mozilla.org/Oxidation
+[Project Quantum]: https://medium.com/mozilla-tech/a-quantum-leap-for-the-web-a3b7174b3c12
+
+[GNOME], a free and open source desktop environment for Linux, went from
+experimenting with Rust in [librsvg] in October 2016 to a [hackfest] in March
+to work on the interoperability between GNOME and Rust to enable more GNOME
+components to be written in Rust. The hackfest participants made good progress,
+be sure to check out the reports at the bottom of the hackfest page for all the
+details. We're all excited about the possibilities of Rust and GNOME working
+together.
+
+[GNOME]: https://www.gnome.org/
+[librsvg]: https://people.gnome.org/~federico/news-2016-10.html#25
+[hackfest]: https://wiki.gnome.org/Hackfests/Rust2017
+
+This year, [npm] started using Rust in production to serve JavaScript packages,
+using Rust to eliminate performance bottlenecks in their platform that serves
+around 350 million packages a day. [Ashley Williams recently gave a
+talk][ag_dubs] at RustFest in Ukraine about npm's experience with Rust in
+production; video is coming soon.
+
+[npm]: https://www.npmjs.com/
+[ag_dubs]: http://2017.rustfest.eu/talks/#how-i-convinced-the-world-s-largest-package-manager-to-use-rust-and-so-can-you
+
+## Rust in community
+
+Speaking of conferences, We've had four Rust conferences in the last year:
+
+- September 9-10, 2016: [RustConf 2016] in Portland, OR, USA;
+- September 17, 2016: [RustFest 2016] in Berlin, Germany;
+- October 27-28, 2016: [Rust Belt Rust 2016] in Pittsburgh, PA, USA;
+- April 29-30, 2017: [RustFest 2017] in Kyiv, Ukraine.
+
+[RustConf 2016]: TODO ATURON PROMISED
+[RustFest 2016]: http://2016.rustfest.eu/
+[Rust Belt Rust 2016]: http://conf2016.rust-belt-rust.com/
+[Rustfest 2017]: http://2017.rustfest.eu/
+
+And we have at least three conferences coming up!
+
+- August 18-19, 2017: [RustConf 2017] in Portland, OR, USA;
+- September, 2017: [Another RustFest] in Zurich, Switzerland;
+- October 26-27, 2017: [Rust Belt Rust 2017] in Columbus, OH, USA.
+
+[Rust Belt Rust 2017]: http://conf2017.rust-belt-rust.com/
+[RustConf 2017]: http://rustconf.com/
+[Another RustFest]: https://rustfest.ch/
+
+That's not even including the [103 meetups worldwide][meetup] about Rust. Will
+you be the one to run the fourth conference or start the 104th meetup? [Contact
+the community team] for help and support!
+
+[meetup]: http://rust.meetup.com/
+[Contact the community team]: https://community.rs/
+
+## Rust in 2017
+
+The [2017 Roadmap] goals have been great for focusing community efforts towards
+the most pressing issues facing Rust today. Of course we'd love for every
+aspect of Rust to improve all the time, but we don't have an infinite number of
+contributors with an infinite amount of time available yet!
+
+Let's check in on some of the initiatives in each of the goals in the roadmap.
+Take a look at each goal's tracking issue for even more initiatives than we're
```

Comment:
How about: "The linked tracking issues give even more detail than the summary here".